src/transactions/OfferTests.cpp

```diff
@@ -572,6 +590,17 @@ TEST_CASE("create offer", "[tx][offers]")
                         auto actualPayment = a1IDrs - delta;
                         checkCrossed(b1, actualPayment, offerAmount);
                     });
+
+                    for_versions_from(10, *app, [&]() {
+                        REQUIRE_THROWS_AS(
```

Comment:
this is not what this test is trying to test:
test ensures that we compute reserve as if creating the offer.
I think that, with liabilities, there should be both a check that verifies that creating an offer with exactly `actualPayment` works, but trying to sell more (by 1 stroop) fails.